components/dropbox/actions/list-file-folders-in-a-folder/list-file-folders-in-a-folder.mjs

```diff
@@ -0,0 +1,83 @@
+import dropbox from "../../dropbox.app.mjs";
+import get from "lodash/get.js";
+
+export default {
+  name: "List All Files/Subfolders in a Folder",
+  description: "Searches for files and folders by name [See the docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesSearchV2__anchor)",
+  key: "dropbox-list-file-folders-in-a-folder",
```

Comment:
The description should be:
`description: "Retrieves a list of files or subfolders in a specified folder [See the docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesListFolder__anchor)",`